include/rive/shapes/mesh.hpp

```diff
@@ -1,20 +1,28 @@
 #ifndef _RIVE_MESH_HPP_
 #define _RIVE_MESH_HPP_
 #include "rive/generated/shapes/mesh_base.hpp"
+#include "rive/span.hpp"
+#include "rive/refcnt.hpp"
 
 namespace rive {
     class MeshVertex;
+
     class Mesh : public MeshBase {
+
     protected:
+        class IndexBuffer : public std::vector<uint16_t>, public RefCnt {};
```

Comment:
@mikerreed I had reservations about this, but I also liked the brevity. Let me know if you need some bleach for your eyes.